c2corg_ui/templates/i18n.html

```diff
@@ -535,3 +535,14 @@
 ## L# tags
 <span translate translate-comment="short for a rope length / pitch">L</span>
 <span translate translate-comment="short for a rappel/abseil">R</span>
+
+## elevation profile
+<span translate translate-comment="Elevation title for mouseover in elevation profile">Elevation</span>
+<span translate translate-comment="Legend in elevation profile">Elevation (m)</span>
+<span translate translate-comment="Elevation unit for mouseover in elevation profile">meters</span>
+<span translate translate-comment="Distance title for mouseover in elevation profile">Distance</span>
+<span translate translate-comment="Legend in elevation profile">Distance (km)</span>
+<span translate translate-comment="Distance unit for mouseover in elevation profile">kilometers</span>
+<span translate translate-comment="Time title for mouseover in elevation profile">Time</span>
+<span translate translate-comment="Duration title for mouseover in elevation profile">Duration</span>
+<span translate translate-comment="Alternate legend in elevation profile">Duration (hrs)</span>
```

Comment:
Perhaps we don't need those "fake" translate strings since they are already made available in the JS directive file (I think the extractor extracts strings in ``translate()`` functions). Not sure it is possible to have "translate-comments" in the JS code but do we really need them here? The strings are quite straightforward.

By the way, do you really need to have almost duplicate strings such as "Elevation" / "Elevation (m)"? Isn't it possible to have the same string for both locations? You could also check in Transifex that some similar strings do not already exist. For instance "elevation" is already available.